types/roslib/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for roslib.js 1.1.0
+// Type definitions for roslib.js 1.1.1
```

Comment:
There doesn't appear to be a 1.1.1.  Note that you don't bump this version number for changes to the type definitions.

Reply:
Sorry! I've reverted this change.